apps/run.go

```diff
@@ -182,7 +187,7 @@ func Run(opts RunOpts) error {
 		for {
 			select {
 			case s := <-signals:
-				run.HandleSignal(s, socket, ctx.attachURL)
+				run.HandleSignal(c, s, socket, ctx.attachURL)
```

Comment:
Are you sure the Scalingo client can be shared by mutliple goroutines?

```suggestion
				run.HandleSignal(ctx.scalingoClient, s, socket, ctx.attachURL)
```